js/jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+};
```

Comment:
not needed, this is a per package config as some will be node, some will be jsdom